src/components/FilmsList.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import "./FilmsList.css";
+
+const films = [
```

Comment:
Add this const to App.js and pass it to your <FilmsList> component. It's mentioned in task. The component should get props(parametr) as array of objects.